java-checks-test-sources/src/main/java/checks/spring/s4605/springBootApplication/thirdApp/controller/Controller.java

```diff
@@ -0,0 +1,6 @@
+package checks.spring.s4605.springBootApplication.thirdApp.controller;
+
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class Controller { } // // Noncompliant [[sc=14;ec=24]] {{'Controller' is not reachable by @ComponentsScan or @SpringBootApplication. Either move it to a package configured in @ComponentsScan or update your @ComponentsScan configuration.}}
```

Comment:
```suggestion
public class Controller { } // Noncompliant [[sc=14;ec=24]] {{'Controller' is not reachable by @ComponentsScan or @SpringBootApplication. Either move it to a package configured in @ComponentsScan or update your @ComponentsScan configuration.}}
```
It does not break the test, just to clean it.